.github/workflows/unit_test.yml

```diff
@@ -11,7 +11,7 @@ env:
 
 jobs:
   test:
-    runs-on: ubuntu-latest-unit-tester
+    runs-on: ubuntu-latest
```

Comment:
Oh, why was this changed? It will be 8x slower if we don't use the unit tester machine for running tests

Reply:
We reverted the changes made to the file for testing purposes in our repository.